src/update/UpdateManager.cpp

```diff
@@ -0,0 +1,150 @@
+/*!********************************************************************
+ Audacity: A Digital Audio Editor
+
+ @file UpdateManager.cpp
+ @brief Declare a class that managing of updates.
+
+ Anton Gerasimov
+ **********************************************************************/
+
+#include "UpdateManager.h"
+#include "UpdatePopupDialog.h"
+
+#include "NetworkManager.h"
+#include "IResponse.h"
+#include "Request.h"
+
+#include "widgets/AudacityMessageBox.h"
+
+#include <wx/platinfo.h>
+#include <wx/utils.h>
+
+static const char* prefsUpdatePopupDialogShown = "/Update/UpdatePopupDialogShown";
+static const char* prefsUpdateScheduledTime = "/Update/UpdateScheduledTime";
```

Comment:
Not very important, I would prefer to simplify with a static IntSetting object... but 32 bits may not be enough and it might be one of the 32 bit Windows builds we still do.

Maybe I should define LongSetting?

Reply:
Yes, if it possible.